auth/maintenance_port_role_manager.cc

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "auth/maintenance_port_role_manager.hh"
+
+#include <seastar/core/future.hh>
+#include <stdexcept>
+#include "log.hh"
+#include "utils/class_registrator.hh"
+
+namespace auth {
+
+constexpr std::string_view maintenance_port_role_manager_name = "com.scylladb.auth.MaintenanceModeRoleManager";
```

Comment:
"MaintenanceMode" appeared here. Should be `MaintenancePortRoleManager`. Or `MaintenanceSocketRoleManager` if we go with "socket" instead of "port" and other places are renamed too.